lib/github_changelog_generator/generator/section.rb

```diff
@@ -0,0 +1,29 @@
+module GitHubChangelogGenerator
+  class Section
+    attr_accessor :name, :prefix, :issues, :labels
+
+    def initialize(opts = {})
+      @name = opts[:name]
+      @prefix = opts[:prefix]
+      @labels = opts[:labels] || []
+      @issues = opts[:issues] || []
+    end
+
+    # @param [Array] issues List of issues on sub-section
+    # @param [String] prefix Name of sub-section
+    # @return [String] Generate section content
+    def generate_content(issues = @issues, prefix = @prefix)
+      content = ""
+
+      if issues.any?
+        content += "#{prefix}\n\n" unless @options[:simple_list]
```

Comment:
The `@options` hash is scattered around the various classes and passed around as needed but never modified once parsed. It would be nice if the options were centrally accessible instead of passing them around, but that is out of scope for this PR.

Instead, I'll just make `simple_list` an invocation option to be removed later.

Reply:
Eh, there are several more options needed by the `Section` class. I guess I'll pass `@options` through and it can be refactored in another PR.